pkg/codegen/python/gen.go

```diff
@@ -1049,9 +1052,11 @@ func (mod *modContext) genAwaitableType(w io.Writer, obj *schema.ObjectType) str
 	fmt.Fprintf(w, "    def __await__(self):\n")
 	fmt.Fprintf(w, "        if False:\n")
 	fmt.Fprintf(w, "            yield self\n")
-	fmt.Fprintf(w, "        return %s(\n", baseName)
+	fmt.Fprintf(w, "        return %s(", baseName)
 	for i, prop := range obj.Properties {
-		if i > 0 {
+		if i == 0 {
+			fmt.Fprintf(w, "\n")
+		} else {
 			fmt.Fprintf(w, ",\n")
 		}
 		pname := PyName(prop.Name)
```

Comment:
Nit: Its really hard to notice the `,`. I think this is much easier to understand. 
```go
                if i > 0 {
			fmt.Fprintf(w, ",")
		}
		pname := PyName(prop.Name)
		fmt.Fprintf(w, "\n            %s=self.%s", pname, pname)
```